src/base/libmesh.C

```diff
@@ -276,6 +276,23 @@ std::terminate_handler old_terminate_handler;
 
 void libmesh_terminate_handler()
 {
+  // If we have an active exception, it may have an error message that
+  // we should print.
+  std::cout << "libMesh terminating";
```

Comment:
Should we regard this as `out` instead of `err`?

Reply:
My thought had been that we don't need any repetition at this point; having multiple outputs of `what()` can be useful if an error manifests differently on different processors, but the fact that something hit the terminate handler doesn't bear repeating.

But thinking about it more... we might not have hit the terminate handler, on processor 0, we might only have thrown on other ranks.  This ought to be going to `libMesh::err` too.